tlair/src/main/scala/at/forsyte/apalache/tla/lir/TlaType1.scala

```diff
@@ -98,19 +98,71 @@ case object StrT1 extends TlaType1 {
 }
 
 /**
- * An uninterpreted type constant.
+ * <p>An uninterpreted type constant such as PROC_NAME.</p>
+ *
+ * <p>Inside the type checker, this class may also represent references to type aliases. Do not use this feature outside
+ * of the type checker.</p>
```

Comment:
I think Scaladoc infers the markup here

```suggestion
 * An uninterpreted type constant such as PROC_NAME.
 *
 * Inside the type checker, this class may also represent references to type aliases. Do not use this feature outside
 * of the type checker.
```